tests/framework/installer/ceph_installer.go

```diff
@@ -112,6 +112,9 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		return err
 	}
 
+	//give nodes a hostname that is different from its k8s node name to confirm that all the daemons will be initialized properly
+	h.k8shelper.ChangeHostnames()
```

Comment:
Wouldn't it make sense to have a new test? One that tests the normal way node name = hostname label and second with node name != hostname lable.

Reply:
This code path will impact all the tests that launch the operator based on the yaml. The helm tests will not change the hostname label so I think we're covered that way.

Reply:
I do need to make a related change here though because the upgrade tests can't have this set when the 0.8 cluster is started. Although after this fix is backported to 0.8, the upgrade test will work with this.